### DISCUSSION TITLE
No Plugin List

### DISCUSSION THREAD

**Discussion by lesbrunn** [apohl79/audiogridder#534] [Q&A]
Hi,
After installing AG, the plugin connects to the server, but there is no plugin list to select from. There's a folder on the server computer that has all my VST plugins, and they've been scanned by the server. What could the problem be? Thanks.
Best,
Leslie,

**Comment by lesbrunn** [comment]
Sorry guys, I was able to sort it out. AG is a brilliant concept! My initial reaction though, is that the screen capture of the opened plugins isn't very sharp. I also had to remove my much needed display scaling because the UI's of the plugins were so big and blurred. Is there any way around this?

**Reply by emack999** [reply]
You can change the screen capture quality in the Server settings.

![781](https://user-images.githubusercontent.com/81657685/116771032-6fab7c80-aa05-11eb-83a6-db0d18c7f357.jpg)

**Reply by lesbrunn** [reply]
Thanks. It is set to high. Is FFmpeg the best quality?

**Reply by lesbrunn** [reply]
Is anyone using v 1.2? How stable is it? (I do realize it's a beta).

**Reply by lesbrunn** [reply]
Sorry. What I meant was: Which is better, webp or mjpeg?

**Reply by apohl79** [reply]
Use webp if it works for you. I have both codecs configured at similar quality. Webp uses less network bandwidth. MJPEG is only there because it allows the plugin to work without ffmpeg, as it can decode jpeg directly and I had issues with Linux at the beginning.

**Reply by lesbrunn** [reply]
Thanks Andreas.

**Comment by CaBuffalo** [comment]
I have been using 1.2 Beta since it’s released. No major issues happens to me. 

Sandbox is the new great feature to improve the stability which I highly recommend you guys to give it a try.

**Reply by lesbrunn** [reply]
Thanks!
